payments/hide-balance.md

```diff
@@ -0,0 +1,43 @@
+# Fake/Panic Balance
+
+Panic balance is a safety feature that was introduced in a digital bank’s mobile application primarily to protect their customers from police extortion in Nigeria. 
+
+## Apps that make use of it
+
+“[**Kuda bank**](https://kudabank.com/)” is one of the digital bank applications that makes use of the Panic Balance /Fake Balance feature as well as the hide balance. Others banks and wallets make most use of the Hide balance feature.
+
+
+
+## How it works
+
+The following steps describe how a customer would set up and use the panic balance feature on Kuda banks app.
+
+- LoginGo to security then select “private mode”
+- Select “panic balance”
+- Enter a value for the balance you would like to see displayed
+-  Shake device to activate and use subsequently 
+
+
+
+## The reason
+
+For as long as I can remember SARS has been a major problem for Nigerian citizens, most especially the younger generations. From extortion to illegal arrests, to murder, they have further increased the insecurity in the country. 
+Unfortunately, the tech industry has been the major victim of SARS brutality with young techies earning a lot at companies like paystack, Flutterwave or working remotely for companies outside the country.
+Sadly, this isn’t a concept that these officers of the law and a lot of the older generation understand, and because of this, young techies are being likened and mistaken as internet scammers simply because they own laptops, drive good cars, and earn a lot of money. The result of this is that they are illegally arrested, extorted, sometimes forced to pay large amounts of money for bail, made to make false police reports or at worst they are killed because they refuse to adhere to their demands.
+
+
+
+## How designers can protect users
+
+1. Use a password or pin each time a person leaves the app... maybe 10 minutes inactive you’d be logged out. This would be the flow
+  - Login to the app
+  - Uses app for sometime
+  - Leaves app for more than 10 minutes then is logged out
+  - Returns to app
+  -  Login Pin/password is requested to continue the session 
+2. Stronger passwords recommendations
+  - user enters password choice
+  - password doesn't meet strength requirements
+  - app recommends better/stronger passwords
+3. Account authentication (sending a key to a registered phone number)
```

Comment:
So I know this was suggested in context of traditional banking experiences - now we have established that it is not a requirement to provide any personally identifiable information in bitcoin applications this can actually be an attack vector.

- See https://en.wikipedia.org/wiki/SIM_swap_scam